types/react/index.d.ts

```diff
@@ -785,7 +785,7 @@ declare namespace React {
         // Pick would not be sufficient for this. We'd like to avoid unnecessary mapping and need a distributive conditional to support unions.
         // see: https://www.typescriptlang.org/docs/handbook/2/conditional-types.html#distributive-conditional-types
         // https://github.com/Microsoft/TypeScript/issues/28339
-        P extends any ? ('ref' extends keyof P ? Pick<P, Exclude<keyof P, 'ref'>> : P) : P;
+        P extends any ? ('ref' extends keyof P ? Omit<P, 'ref'> : P) : P;
```

Comment:
Was `Omit` not a thing when this line was added?

Reply:
Looks like this was the original commit from Nov 13, 2018: https://github.com/johnslemmer/DefinitelyTyped/commit/5c49d4ddc6f5bd06c5cf04d911398f3db05ead54

Omit wasn't added to typescript until 3.5 on May 29, 2019